solidus_product_feed.gemspec

```diff
@@ -1,38 +1,30 @@
-# encoding: UTF-8
-$:.push File.expand_path('../lib', __FILE__)
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$:.unshift lib unless $:.include?(lib)
+
 require 'solidus_product_feed/version'
 
 Gem::Specification.new do |s|
   s.platform    = Gem::Platform::RUBY
   s.name        = 'solidus_product_feed'
   s.version     = SolidusProductFeed::VERSION
-  s.summary     = 'Solidus extension that provides an RSS feed for products'
-  s.description = 'A Solidus extension that provides an RSS feed for products'
-  s.required_ruby_version = '>= 1.8.7'
+  s.summary     = 'A Spree extension that provides an RSS feed for products'
+  s.description = 'A Spree extension that provides an RSS feed for products, with Google Shopper extensions'
```

Comment:
any reason this is being updated to `Spree` over `Solidus`, something matching upstream?

Reply:
That's it, matching upstream.